aviary/wren/utils.py

```diff
@@ -314,7 +364,8 @@ def count_wyks(aflow_label: str) -> int:
 
 
 def count_params(aflow_label: str) -> int:
```

Comment:
input -> `full_label` again to be consistent. I am not sure sure that `full_label` is a good name. `alfow_label` is strictly just the bit before the colon. Is `aflow_label_with_chemsys` too long?

Reply:
Not too long, I think. I'll change it to `aflow_label_with_chemsys`.